dataflow/build.gradle

```diff
@@ -1,6 +1,7 @@
 dependencies {
     implementation project(':javacutil')
     implementation project(':checker-qual')
+    implementation 'org.plumelib:plume-util:1.1.5'
```

Comment:
This dependency needs to be added to `checker-framework/docs/developer/release/maven-artifacts/dataflow-pom.xml`.  (Once we start using the Maven publish gradle plugin, the pom file will be automatically generated and we won't have to do this.)